awx/ui_next/src/screens/InstanceGroup/InstanceGroupList/InstanceGroupListItem.jsx

```diff
@@ -108,6 +109,13 @@ function InstanceGroupListItem({
                   <b>{instanceGroup.name}</b>
                 </Link>
               </span>
+              {instanceGroup.is_isolated ? (
+                <span css="margin-left: 12px">
+                  <Chip key={instanceGroup.id} isReadOnly>
```

Comment:
@nixocio I think if we used the `import { Label } from '@patternfly/react-core';` component it would more closely align with the mockups.